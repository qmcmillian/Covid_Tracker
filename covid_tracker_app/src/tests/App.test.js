
import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import CountryPicker from '../components/CountryPicker';

describe('CountryPicker', () => {
  it("should render my component", () => {
    const wrapper = shallow(<CountryPicker/>);
    expect(wrapper).toMatchSnapshot();
  });
});