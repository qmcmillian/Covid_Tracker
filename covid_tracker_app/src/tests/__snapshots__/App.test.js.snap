// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CountryPicker should render my component 1`] = `
<WithStyles(ForwardRef(FormControl))
  className="formControl"
>
  <WithStyles(ForwardRef(NativeSelect))
    defaultValue=""
    onChange={[Function]}
  >
    <option
      value=""
    >
      Global
    </option>
  </WithStyles(ForwardRef(NativeSelect))>
</WithStyles(ForwardRef(FormControl))>
`;
